zpeclient.go

```diff
@@ -300,3 +303,39 @@ func (c *Cache) authorize(principal string, check AthenzAccessCheck) (bool, erro
 	}
 	return false, nil
 }
+
+func (c *Cache) checkUpdateTime(configmap string) (res bool, err error) {
+	var timestamp interface{}
+	timestamp, exist, err := c.cmIndexInformer.GetStore().GetByKey(configmap)
+	if err != nil {
+		return false, err
+	}
+	if !exist {
+		return false, fmt.Errorf("unable to get config map athenzcall-config from configmap informer cache")
+	}
+	obj, ok := timestamp.(*corev1.ConfigMap)
+	if !ok {
+		return false, fmt.Errorf("Unable to cast interface to config map object")
+	}
+	c.log.Println("latest contact time is: ", obj.Data["latest_contact"])
+	lastUpdateTime := obj.Data["latest_contact"]
+	if len(lastUpdateTime) > 0 && lastUpdateTime[0] == '"' {
+		lastUpdateTime = lastUpdateTime[1:]
+	}
+	if len(lastUpdateTime) > 0 && lastUpdateTime[len(lastUpdateTime)-1] == '"' {
+		lastUpdateTime = lastUpdateTime[:len(lastUpdateTime)-1]
+	}
```

Comment:
Can we use a string function to remove the quotes? https://golang.org/pkg/strings/#ReplaceAll
Can we also investigate why the syncer is writing to the configmap with quotes? Can we potentially remove it from there instead?